frontend/comp/ResetButton.jsx

```diff
@@ -7,7 +7,7 @@ export default class ResetButton  extends React.Component {
       evt.preventDefault();
       // Resets index file. TODO: Images are still stored!
       blockstack.putFile('index.json')
-  
+      window.location.reload()
```

Comment:
this is a good idea, but doesn't work, since the `putFile` call might not be finished, before reloading the page.
The function actually returns a Promise, so you can add a `.then` function and reload the page in there.

Reply:
oh yes, sure need to wait